ui/base-tool/src/main/java/org/cru/godtools/base/tool/databinding/TextViewAdapters.kt

```diff
@@ -3,9 +3,44 @@ package org.cru.godtools.base.tool.databinding
 import android.widget.TextView
 import androidx.annotation.ColorInt
 import androidx.databinding.BindingAdapter
+import com.squareup.picasso.Picasso
+import org.ccci.gto.android.common.picasso.widget.TextViewDrawableEndTarget
+import org.ccci.gto.android.common.picasso.widget.TextViewDrawableStartTarget
+import org.cru.godtools.base.fileManager
 import org.cru.godtools.base.tool.model.view.bindTo
+import org.cru.godtools.xml.model.Resource
 import org.cru.godtools.xml.model.Text
 
 @BindingAdapter("android:text", "android:textSize", "defaultTextColor", requireAll = false)
 fun TextView.bindTextNode(text: Text?, textSize: Float?, @ColorInt defaultTextColor: Int?) =
     text.bindTo(this, textSize, defaultTextColor)
+
+@BindingAdapter("android:drawableStart", "startImageSize")
```

Comment:
I would rather name the binding attribute `drawableStartSize`

Reply:
that way in the view layout it's easier to see the correlation between `drawableStart` and `drawableStartSize`